presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -519,6 +519,14 @@ public void testSTStartEndPoint()
         assertInvalidFunction("ST_AsText(ST_EndPoint(ST_GeometryFromText('POLYGON ((2 0, 2 1, 3 1))')))", "ST_EndPoint only applies to LINE_STRING. Input type is: POLYGON");
     }
 
+    @Test
+    public void testSTPoints()
+    {
+        assertFunction("transform(ST_Points(ST_GeometryFromText('LINESTRING (8 4)')), x -> ST_AsText(x))", new ArrayType(VARCHAR), ImmutableList.of("POINT (8 4)", "POINT (8 4)"));
+        assertFunction("transform(ST_Points(ST_GeometryFromText('LINESTRING (8 4, 3 9, 5 6)')), x -> ST_AsText(x))", new ArrayType(VARCHAR), ImmutableList.of("POINT (8 4)", "POINT (3 9)", "POINT (5 6)"));
```

Comment:
nit: consider extracting a helper method similar to `assertSTGeometries`:

`assertSTPoints("LINESTRING (8 4, 3 9, 5 6)", "8 4", "3 9", "5 6")`